x-pack/plugins/cloud_security_posture/server/saved_objects/mappings.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SavedObjectsTypeMappingDefinition } from '@kbn/core/server';
+
+export const cspRule: SavedObjectsTypeMappingDefinition = {
```

Comment:
Consider naming the exported constants here more verbosely to reflect their content, e.g. `cspRuleSavedObjectMapping`

Reply:
will do